### DISCUSSION TITLE
Is there a way to give confidence intervals for the observed function y with a Student T likelihood?

### DISCUSSION THREAD

**Discussion by ghost** [cornellius-gp/gpytorch#1837] [Q&A] [answered]
When using the Student T likelihood is there a way to get a confidence interval for the observed posterior (I mean the non-latent one you get from calling likelihood(model(test_x)) )? I guess I could constrain the nu parameter to be strictly larger than 2  and do it over the standard deviation, but I was wondering if there is another way.

Maybe I could add the noise std to the variational one ? Would that make sense?

Thanks for any answer

**Comment by gpleiss** [comment, marked as answer]
There isn't a good way to compute the confidence interval, because the student t likelihood is non-conjugate to the GP variational distribution. You cannot compute a closed-form version of the marginal predictive distribution.

**Reply by ghost** [reply]
would the standard deviation of the GP variational distribution work as an approximation? I thought about it again and now I'm not sure if the variational distribution includes the noise variance or not. In my orginal question I assumed it did not and that the variational distribution only attempted to approxiamte the latent function without noise.

**Reply by gpleiss** [reply]
That would give you a confidence interval for the latent posterior, but not for the (observed) posterior predictive.
